src/util/name_table.cpp

```diff
@@ -1,64 +1,58 @@
 #include "util/name_table.hpp"
 #include "storage/io.hpp"
-#include "util/exception.hpp"
 #include "util/log.hpp"
 
-#include <algorithm>
-#include <iterator>
-#include <limits>
-
-#include <boost/filesystem/fstream.hpp>
-
 namespace osrm
 {
 namespace util
 {
 
-NameTable::NameTable(const std::string &filename)
+NameTable::NameTable(const std::string &file_name)
```

Comment:
It is still a table with block index and variable part of the index that  prepends variable data. I would keep `NameTable` name here